src/index.tsx

```diff
@@ -23,6 +23,13 @@ type ChartData = {
   end?: Date;
 };
 
+// Event contains label and value of current datapoint
+interface SelectEvent {
```

Comment:
Let's use `type` or update the others to `interface` to keep things consistent

Reply:
yes, changing it